internal/outputer/formatter/policies_content.go

```diff
@@ -34,6 +34,24 @@ func newPoliciesContent(pf policiesFormatter, colorizer colorizer) *policiesCont
 	}
 }
 
+func (pc *policiesContent) FormatPolicy(output *scheme.Flattened, policyName string) []byte {
+	pc.sb.Reset()
+
+	if _, ok := output.AsOrderedMap().Get(policyName); ok {
```

Comment:
I think it'd be better to change to:
```
	if _, ok := output.AsOrderedMap().Get(policyName); !ok {
		return nil
	}
	pc.sb.Reset()
	... rest of the code
```

that way we won't return a seemingly valid (empty) array when the policy doesn't exist